package.json

```diff
@@ -1,11 +1,11 @@
 {
   "private": true,
-  "name": "rtvi-client-web",
+  "name": "pipecat-client-web",
   "version": "0.2.2",
   "workspaces": [
-    "rtvi-sandbox",
-    "rtvi-client-js",
-    "rtvi-client-react"
+    "pipecat-sandbox",
```

Comment:
The sandbox doesn't exist anymore and can be removed here.

Reply:
Thanks!